CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 project(libcrashreporter-qt)
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
3.6 would suffice for `IMPORTED_TARGET`. Any other reasons why you bumped it to 3.12?